test/skimNtuple2016.cpp

```diff
@@ -3583,7 +3960,7 @@ int main (int argc, char** argv)
             theSmallTree.m_HHkinsvfit_m   = tlv_HHsvfit.M () ;
           } // in case the SVFIT mass is calculated
 
-          // raw kinfit TES up
+          /*// raw kinfit TES up
```

Comment:
please do not use `\*...*\` for comments, put `\\` in front of each line, it' easier to read and correct in git.